### PR TITLE
Add the jest test files

### PR DESCRIPTION
**Pull request by Marcraphael12**
In this PR, we are testing our app using jest by following those requirements

- Set up React Testing Library
- Write unit tests for the files operate.js and calculate.js using Jest.
- Create unit tests for all React components
Happy reviewing!!!

### CODE REVIEW

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 🔴 🛑
Hello @Marcraphael12, great job you are doing so far in this project. 💯
TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️

- [x] Kindly fix the `app.test.js` that is currently failing. 

Capture
Capture1


Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

cat


**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Quote.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-unused-vars */
+import renderer from 'react-test-renderer';
```
Comment (rloterh) [inline src/components/Quote.test.js:2]: - [x] Test cases are still failing. Kindly ensure to improve upon this aspect.

**Failing test case:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Team, 

You did great on this project, and for making the changes requested by the previous reviewer! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below this review. Kindly take your time to correctly address those issues outlined in the previous code reviews, and please check carefully to ensure that the changes are working as intended._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Quote.test.js (hunk 1 above)
Comment (Marcraphael12) [reply src/components/Quote.test.js:2]: Finally fixed

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @Marcraphael12 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**